dvc/remote/ssh/__init__.py

```diff
@@ -101,7 +96,12 @@ def _try_get_ssh_config_keyfile(user_ssh_config):
             return identity_file[0]
         return None
 
-    def ssh(self, host=None, user=None, port=None, **kwargs):
+    def ssh(self, path_info=None):
+        if path_info:
+            host, user, port = path_info.host, path_info.user, path_info.port
+        else:
+            host, user, port = self.host, self.user, self.port
```

Comment:
We didn't have this logic before though, why do we need it now?

Reply:
Ah, because we've removed `self.path_info`?

Reply:
Yes

Reply:
@pared We now have path_info back, should we remove this `else` now?